docs/elements/postgresql/faq.md

```diff
@@ -10,10 +10,20 @@ parent: Back to Element Guides
 order: 50
 ---
 
-## FAQ
```

Comment:
Can you change the file name to tips.md, and in the heading area, change `SEO`, `title`, and `description` to Tips.